process/Issue_Triaging.md

```diff
@@ -0,0 +1,113 @@
+# Issue Triaging
+
+Triaging an issue is a multi-step process that is collaboratively performed by community. Triaging an issue usually takes around one business day but may take longer, for example, when the reviewer is not around. Goal of triaging is to provide you with a clear understanding of what will happen to your issue. For example, after your feature request was triaged you know whether we plan to tackle the issue or whether we'll wait to hear what the broader community thinks about this request.
+
+
+## Terminologies
+
+* TSC: Technical Steering Committee. An informal group of people who meet on a monthly basis and discuss technical topics related to software development. Everyone is welcome to join the Technical Steering Committee. Refer to the charter for more details.
+
+
+## Process
+
+The basic process of triaging issues is
+
+|First Step     |Next Step            |Result     |
+|---            |---                  |---        |
+|TSC review ->  |Accepted, or         |
+|               |Community review ->  |Closed, or |
+|               |                     |Accepted   |
+
+Please note:
+1. TSC reviewers can accept an issue but cannot close an issue before it is being reviewed by the community (the "Community Review" step).
+2. We are much less likely to accept PRs that have no associated accepted issues
```

Comment:
I see no reason why PRs would not be accepted if they are reasonable. Maybe we should reword this to say that it's good to open issues if you plan to work on something to make sure you get feedback in advance and to avoid duplicating work.

Reply:
I would expect to see an issue in order to define a problem statement and a PR to implement the solution. Having said that, if this is small a PR in a lot of cases is enough.